config.lua

```diff
@@ -809,6 +809,18 @@ lspconfig.vimls = add_lsp {
   verbose = false
 }
 
+---# V
+--- __Status__: Works
+--- __Site__: https://github.com/vlang/v-analyzer
+--- __Installation__: https://github.com/vlang/v-analyzer?tab=readme-ov-file#installation
+lspconfig.v_analyzer = add_lsp {
+  name = "v_analyzer",
+  language = "v",
+  file_patterns = { "%.vv?$", "%.vsh$" },
```

Comment:
Should these patterns be added to `vls` too?

Reply:
I think so, but I just read that [vls](https://github.com/vlang/vls) is no longer officially maintained as the official V lsp server, so I think I could update the description.
What do you think?

Reply:
Yep, go for it.

Reply:
Ok, I'm on it.